src/main/resources/org/jenkinsci/plugins/workflow/libs/SCMRetriever/config.jelly

```diff
@@ -25,5 +25,8 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
-    <f:descriptorRadioList varName="scm" instance="${instance.scm}" descriptors="${descriptor.SCMDescriptors}" title="${%Source Code Management}"/>
+    <f:dropdownDescriptorSelector field="scm" descriptors="${descriptor.SCMDescriptors}" title="${%Source Code Management}"/>
```

Comment:
This change (same for `SCMSourceRetriever`) has a non-trivial effect on the layout (changes it from radio buttons to a dropdown), but I think we have to make this change or else the layout with the new field is really confusing. We should do some interactive testing to make sure everything still works correctly. I am not sure if there was a reason that radio buttons were used in the first place, maybe it was to reduce the level of nesting?